data/scenarios/04Testing/428-drowning-destroy.yaml

```diff
@@ -0,0 +1,46 @@
+name: Drowning results in destruction
+description: |
+  Destroying robots doesn't work.
+  https://github.com/swarm-game/swarm/issues/428
+win: |
+  try {
+    _ <- robotNamed "bot";
+    return false
+  } { return true }
+
+solution: |
+  log "start";
+  wait 5;
+robots:
+  - name: bot
```

Comment:
I would suggest making the map 2x2 and having both base and bot attempt to go into the water.
Then check that base still lives and the bot does not.